scripts/release.kts

```diff
@@ -19,7 +19,7 @@
  * To execute the script two environment variables that are used by Maven have to be present: BINTRAY_USERNAME, BINTRAY_API_KEY
  *
  * Example usage (assuming that the script is executed from the `<project-root>/scripts` directory and the distribution files are in `<project-root>/dist`):
- * $ kotlinc -script release.kts -- -d ../dist  -javaRepositoryUrl https://api.bintray.com/maven/sentry/sentry-java/sentry-java/ -androidRepositoryUrl https://api.bintray.com/maven/sentry/sentry-android/sentry-java/ | sh
+ * $ kotlinc -script release.kts -- -d ../dist | sh
```

Comment:
do we still need this if using `com.vanniktech.maven.publish`?

Reply:
If we want to use craft and publish to Maven as a separate step, yes.